client/analytics/ad-tracking.js

```diff
@@ -1,57 +1,54 @@
 /**
  * External dependencies
  */
-const async = require( 'async' ),
-	noop = require( 'lodash/utility/noop' ),
-	map = require( 'lodash/collection/map' ),
-	some = require( 'lodash/collection/some' ),
-	debug = require( 'debug' )( 'calypso:ad-tracking' );
+import async from 'async';
+import noop from 'lodash/utility/noop';
+import map from 'lodash/collection/map';
+import some from 'lodash/collection/some';
+import debugFactory from 'debug';
```

Comment:
I'm happy to change it if it's an improvement, but I think we should add this to the docs/linting rules. What do you think about adding `sort-imports` ([docs](https://github.com/eslint/eslint/blob/master/docs/rules/sort-imports.md)) to our ESLint config?